File: cpp/problems/syntax/syntax021.cpp
#include <iostream>
using namespace std;

int main(int argc, char *argv[]) {
    int a[] = {1, 3, 7, 4};
    for (int i = 0; i < 4; i++) cout << a[i] << " ";
    cout << endl;
    for (int i = 0; i < 4; i++) cout << i[a] << " ";
    cout << endl;
    for (int i = 0; i < 4; i++) cout << a + i << " ";
    cout << endl;
    for (int i = 0; i < 4; i++) cout << *(a + i) << " ";
    cout << endl;
    cout << a << endl;
    cout << *a << endl;
    return 0;
}

File: objective-c/syntax/syntax021.m
#include <Foundation/Foundation.h>

int main(int argc, char *argv[]) {
    NSString *names[3];
    names[0] = @"Belfast";
    names[1] = @"Copenhaga";
    names[2] = @"Dwashington";
    NSArray *cities = [NSArray arrayWithObjects : names count : 3];
    NSLog(@"%@", cities);
    return 0;
}
